test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env rspec
```

Comment:
one think that I notice now...why so complicated structure of test directory? I think simple `test/AutoinstallAskInclude_test.rb` is enough, or if you really want to separate it, then `test/include/..` but two levels seem too much for me.